README.md

```diff
@@ -57,6 +57,19 @@ There is a 10MB limit on arXiv submissions, so to make it fit:
     You can whitelist some PDFs to skip the global size using
     `images_whitelist`.
 
+#### TikZ picture source code concealment
+
+To prevent the upload of carefully crafted tikzpicture source code or raw simulation
```

Comment:
```suggestion
To prevent the upload of tikzpicture source code or raw simulation
```
I'd remove 'carefully crafted' as the tone implies that one does not want to share something just because it took time to create :)
I know it's part of your spirit, but let's leave it more vague here, as there might be many other reasons people cannot share their tikz code.

Reply:
sure ;-)